plugins/factory-plugin/package.json

```diff
@@ -53,5 +53,27 @@
             "json"
         ],
         "testURL": "http://localhost/"
+    },
+    "contributes": {
+        "menus": {
+            "editor/context": [
+                {
+                    "command": "factory-plugin.create-workspace",
+                    "when": "resourceFilename =~ /devfile/"
```

Comment:
AFAIK now the devfile is named `.devfile.yml` no ?

Reply:
All names containing 'devfile' are suitable.
E.g. `devfile`, `.devfile`, `devfile.yaml` and others.
But this mask is not case sensitive, so `Devfile` will not be handled.